### ISSUE TITLE
The download png action is too slow and produces clipped results

### ISSUE THREAD

**Issue by abhcs** [PolicyEngine/policyengine-app#966]
On saving the result on https://policyengine.org/us/policy?focus=policyOutput.netIncome&reform=41392&region=us&timePeriod=2023&baseline=2 as a png, I have to wait a few seconds to get:

![budgetaryImpactPolicy #41392](https://github.com/PolicyEngine/policyengine-app/assets/31144719/e1b07825-0349-4686-85bb-e35341f91d33)

Note that the right side of the chart is clipped: on the website, there is more space beside the second bar.

Expected: less latency, full unclipped chart

**Comment by abhcs** [comment]
Yeah, on Chrome I get the png immediately. On Safari (my default), I have to wait for about 5 seconds for the png to get generated. The delay is probably related to the async call here:

 https://github.com/PolicyEngine/policyengine-app/blob/818f3f5d6024b32b7efa566750d9dd3986af6111/src/pages/policy/output/utils.js#L24

**Comment by abhcs** [comment]
Plotly js has its own [image export functionality](https://plotly.com/javascript/static-image-export/). Composing this with the title and description may yield a more stable solution. It may be a bit tricky to get this export functionality to work with react-plotly.

Until we have fixed this, we could disable "download png" -- just a thought.

**Comment by abhcs** [comment]
Using Plotly js's "download png" functionality on the examples on [this page](https://plotly.com/javascript/figure-labels/) I see no delay on Safari or Chrome.

**Comment by anth-volk** [comment]
@MaxGhenis What are your thoughts on disabling PNG downloads? Currently, you can download PNGs, but they sometimes have the wonky formatting present in the photo in this and #1032.

Otherwise, @abhcs, I think you're probably right, it'd be better to simply compose a custom output that was standardized across platforms, charts, inputs, etc. Part of the challenge of what we're doing right now is how Plotly renders the chart, defining a fixed-position div, then placing the chart within the div without actually resizing it, making all other formatting quite difficult.

**Comment by MaxGhenis** [comment]
Given the ease of screenshotting I'm inclined to only allow png downloads if it works really well

**Comment by anth-volk** [comment]
In that case @abhcs, could you go ahead and disable, then open a PR here?

**Comment by MaxGhenis** [comment]
Thank you for filing this issue. Closing as part of the `policyengine-app` (v1) wind-down: policyengine.org is now served by [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2), and this repository is being archived, so this issue is frozen against a codebase that no longer deploys anywhere.

If this is still relevant on the current site, please re-file it against [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2) — we'd welcome it there. Apologies that this sat open through the migration.
